osu.Game/Screens/Edit/Editor.cs

```diff
@@ -181,12 +182,20 @@ private void onModeChanged(EditorScreenMode mode)
             LoadComponentAsync(currentScreen, screenContainer.Add);
         }
 
+        private double scrollAccumulation;
+
         protected override bool OnScroll(ScrollEvent e)
         {
-            if (e.ScrollDelta.X + e.ScrollDelta.Y > 0)
+            scrollAccumulation += e.ScrollDelta.X + e.ScrollDelta.Y * (e.IsPrecise ? 0.1 : 1);
+            if (Math.Abs(scrollAccumulation) < 1)
+                return true;
+
+            if (scrollAccumulation > 0)
                 clock.SeekBackward(!clock.IsRunning);
             else
                 clock.SeekForward(!clock.IsRunning);
+
+            scrollAccumulation = 0;
```

Comment:
Should be something like:

```
while (scrollAccumulation-- > 0)
{
    ...
}
```

I believe, i.e. if precise scroll of 2.5, should seek forward 2 times.

Reply:
Ofc make sure it doesn't go negative >_>